src/EditorFeatures/CSharpTest/Completion/LoadDirectiveCompletionProviderTests.cs

```diff
@@ -19,7 +19,7 @@ public LoadDirectiveCompletionProviderTests(CSharpTestWorkspaceFixture workspace
         {
         }
 
-        internal override CompletionListProvider CreateCompletionProvider()
+        internal override CompletionProvider CreateCompletionProvider()
```

Comment:
1) I think "CompletionListProvider" is a better name as these are providers of "CompletionList"s.
2) If you change this name back, it's likely that thsi review can be totally visible within GitHub.